rusk-uplink/src/definitions.rs

```diff
@@ -274,6 +278,41 @@ impl<'a> RawTransaction<'a> {
     }
 }
 
+#[derive(Debug, Default)]
+pub struct RawEvent<'a> {
```

Comment:
Either make data a slice with a lifetime, or make name a string, having both makes little sense

Reply:
Or maybe even use a 'static str for the name

Reply:
Went with taking a `String`.